components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -871,6 +872,25 @@ class SweepInterval(BaseModel):
     search_result: int = Field(default=30, gt=0)
 
 
+class McpServer(BaseModel):
+    DEFAULT_PORT: ClassVar[int] = 8000
+
+    host: str = "localhost"
+    port: int = DEFAULT_PORT
+
+    @field_validator("host")
+    @classmethod
+    def validate_host(cls, value):
+        _validate_host(cls, value)
+        return value
+
+    @field_validator("port")
+    @classmethod
+    def validate_port(cls, value):
+        _validate_port(cls, value)
+        return value
```

Comment:
these can be removed if you sync the latest code and mark `host` as `DomainStr` type, and `port` as `Port` type